src/C/ogr_srs_api.jl

```diff
@@ -416,30 +376,55 @@ end
 
 Convert coordinate system to ERMapper format.
 """
-function OSRExportToERM(arg1::OGRSpatialReferenceH, arg2, arg3, arg4)
+function OSRExportToERM(arg1, arg2, arg3, arg4)
     ccall((:OSRExportToERM, libgdal), OGRErr, (OGRSpatialReferenceH, Cstring, Cstring, Cstring), arg1, arg2, arg3, arg4)
 end
 
-
 """
-    OSRMorphToESRI(OGRSpatialReferenceH) -> OGRErr
+    OSRMorphToESRI(OGRSpatialReferenceH hSRS) -> OGRErr
 
 Convert in place to ESRI WKT format.
 """
-function OSRMorphToESRI(arg1::OGRSpatialReferenceH)
+function OSRMorphToESRI(arg1)
     ccall((:OSRMorphToESRI, libgdal), OGRErr, (OGRSpatialReferenceH,), arg1)
 end
 
-
 """
-    OSRMorphFromESRI(OGRSpatialReferenceH) -> OGRErr
+    OSRMorphFromESRI(OGRSpatialReferenceH hSRS) -> OGRErr
 
 Convert in place from ESRI WKT format.
 """
-function OSRMorphFromESRI(arg1::OGRSpatialReferenceH)
+function OSRMorphFromESRI(arg1)
     ccall((:OSRMorphFromESRI, libgdal), OGRErr, (OGRSpatialReferenceH,), arg1)
 end
 
+"""
+    OSRConvertToOtherProjection(OGRSpatialReferenceH hSRS,
+                                const char * pszTargetProjection,
+                                const char *const * papszOptions) -> OGRSpatialReferenceH
+
+Convert to another equivalent projection.
+
+### Parameters
+* **hSRS**: source SRS
+* **pszTargetProjection**: target projection.
+* **papszOptions**: lists of options. None supported currently.
+
+### Returns
+a new SRS, or NULL in case of error.
+"""
+function OSRConvertToOtherProjection(hSRS, pszTargetProjection, papszOptions)
+    ccall((:OSRConvertToOtherProjection, libgdal), OGRSpatialReferenceH, (OGRSpatialReferenceH, Cstring, Ptr{Cstring}), hSRS, pszTargetProjection, papszOptions)
```

Comment:
I wonder where we can find example code for new functions being introduced haha. Always takes me a long time to figure out a working example from scratch

Reply:
not a blocker for this PR though.

Reply:
Yep I agree, don't know. This one is new since 2.3 apparently: https://gdal.org/api/ogr_srs_api.html?highlight=osrconverttootherprojection#_CPPv427OSRConvertToOtherProjection20OGRSpatialReferenceHPKcPPCKc